src/pages/index.tsx

```diff
@@ -71,28 +71,6 @@ export default function Index(): JSX.Element {
           <NodeFeature img={featureImg} featureText={nodeFeature3} />
         </section>
 
-        <section className="join-node">
```

Comment:
Any CSS that needs to be removed with this one or is the class just for the selector?

Reply:
Good point let me double check.

Reply:
I've removed the unused CSS now.